### ISSUE TITLE
Deprecate `mongodb-memory-server-global-3.6`

### ISSUE THREAD

**Issue by hasezoey** [nodkz/mongodb-memory-server#559]
Discussion to deprecate package `mongodb-memory-server-global-3.6`, [because it does not seem like it is used anymore](https://www.npmjs.com/package/mongodb-memory-server-global-3.6) (in recent versions, like 7.x) and because release [3.6 is EOL](https://docs.mongodb.com/manual/release-notes/#eol-ed-releases) (End-Of-Life)

@nodkz how about deprecating it, like we have done with `mongodb-memory-server-3.4` for release 8.0?
btw, with `mongodb-memory-server-3.4` there is still one version that is not deprecated, `0.0.1-security` (also exists in the other packages)

**Comment by hasezoey** [comment]
> So If we remove it then need to give something instead e.g 5.0 ;)

see #555, it will be the default version for 8.x
but we could make a `4.0` global package, the previous default (default in 6.x and 7.x)

> As a simple solution, we may stop publishing new versions of the 3.6 branch and deprecate it when it will really have some problems (security, binary absence, or something else). Really big enterprises use old and stale software because they are too big and it's too risky to upgrade rather than keep old versions.

yes i know, i asked if it would be OK to stop publishing that version (starting with 8.x) and maybe do the same as with 3.4

> But keeping 3.6 is the best solution. Of course, if it is possible. @hasezoey what do you think?

currently, there would not be much of a problem with keeping 3.6 around, i just looked it up if we should drop it because i saw that the downloads for that package were really low and the 3.6 release is EOL
also, no tests cover that version anymore - so compatibility is not guaranteed (even with 7.x)

---

TL;DR: i would be for stopping to publish new versions for package `mongodb-memory-server-global.3.6` starting with `8.0.0` (but like with `mongodb-memory-server-global.3.4`, it will be kept around with a `old/7.x` branch, if we should ever need to update it)
but it may be appropriate to deprecate the `8.0.0-beta.x` releases for that package

**Comment by nodkz** [comment]
> see #555, it will be the default version for 8.x

v5.0 has minor breaking changes so I don't see that It brings a lot of problems to users. So let 5.0 will be the default for v8.

> TL;DR: i would be for stopping to publish new versions for 3.6

Agree. Let's stop.

**Comment by hasezoey** [comment]
with b75e05453d3d125e16083d77488bbea5e2a37fdc the package `mongodb-memory-server-global-3.6` was removed (from the beta branch and so from 8.x)

to avoid confusion, please deprecate the 8.0 beta version for that package (range: `>8.0.0-beta.0`)

**Comment by nodkz** [comment]
@hasezoey 
v3.6 for all 8.0.0 were deprecated or removed
v4.0 was undeprecated
Also, I made you a maintainer for all MMS packages on npm. So now you also may run these commands ;)

Tnx.
